sdcm/utils/sstable/load_utils.py

```diff
@@ -228,3 +244,24 @@ def get_load_test_data_inventory(cls, column_number: int, big_sstable: bool,
             return MULTI_COLUMNS_DATA
 
         return []
+
+    @classmethod
+    def create_keyspace(cls, node, keyspace_name: str = "keyspace1", strategy: str = 'SimpleStrategy',  # pylint: disable=too-many-arguments
+                        replication_factor: int = 1, drop_ks_if_exists: bool = False):
+        if drop_ks_if_exists:
+            node.run_cqlsh(f"DROP KEYSPACE {keyspace_name}")
+
+        node.run_cqlsh("""
+                CREATE KEYSPACE %s WITH replication = {'class': '%s',
```

Comment:
```suggestion
                CREATE KEYSPACE IF NOT EXISTS %s WITH replication = {'class': '%s',
```

would remove the need of having `drop_ks_if_exists`

Reply:
I did it in intention. I want to drop keyspace if it exists.
It should be appropriate schema that selected snapsot needs